app/js/service/rate.js

```diff
@@ -12,7 +12,7 @@
   const USE_CORS_PROXY = RSBP_CONFIG.rate.useCorsProxy;
   const EXPIRATION = RSBP_CONFIG.rate.expiration;
   const IS_BTC = CURRENCY === "BTC";
-  const URL = "https://api.bitcoinaverage.com/ticker/" + CURRENCY + "/last";
+  const URL = "https://apiv2.bitcoinaverage.com/indices/global/ticker/" + CURRENCY + "/last";
```

Comment:
Oh - I think you're looking at the Last pull request, not the current one we're discussing in this pull request:
https://github.com/RagnarDanneskjold/RSBP/blob/master/app/js/service/rate.js

THIS is your new line 15:
  const URL = "https://apiv2.bitcoinaverage.com/indices/global/ticker/" + CURRENCY + "/last";
  and you need to add a new line 36:
         data = data['last'];